go/def.bzl

```diff
@@ -558,7 +574,8 @@ go_library = rule(
 go_binary = rule(
     go_binary_impl,
     attrs = go_library_attrs + _crosstool_attrs + {
-        "stamp": attr.bool(default = False),
+        "stamp": attr.bool(default = True),
```

Comment:
Can we do this if/when we decide we like the design? What's the policy on compatibility? I'd like someone from go team to weigh in on this before publicizing, but it unblocks us now.